### PR TITLE
Add excel viewer to analysis output preview

### PR DESCRIPTION
**Pull request by deepsidhu85**
## Description of changes
What did you change in this pull request?  Provide a description of files changed, user interactions changed, etc.  Include how to test your changes.

Added excel viewer for analysis output files of type xlsx.

## Related issue
Link to the GitHub issue this pull request addresses using the `#issuenum` format.  If it completes an issue, use `Fixes #issuenum` to automatically close the issue.
#354 

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [X] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
~* [ ] Tests added (or description of how to test) for any new features.~
~* [ ] User documentation updated for UI or technical changes.~

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -672,6 +672,13 @@
 			<artifactId>gson</artifactId>
 			<version>${com.google.code.gson.version}</version>
 		</dependency>
+		<!-- https://mvnrepository.com/artifact/com.monitorjbl/xlsx-streamer -->
+		<dependency>
+			<groupId>com.monitorjbl</groupId>
+			<artifactId>xlsx-streamer</artifactId>
+			<version>${com.monitorjbl.xml.streamer.version}</version>
```
Comment (joshsadam) [inline pom.xml:679]: What is the advantages of this over using apache poi which we already use?

Comment (deepsidhu85) [reply pom.xml:679]: Apache POI crashes when creating a workbook if the column data doesn't match the column type. For example the amr detection pipeline outputs an excel file with column types set to `Number` but the data is `Text`. After discussing this with Tom I found this package to help us :)

Comment (joshsadam) [reply pom.xml:679]: Perfect, just wanted to know, Thanks.

**Comment by deepsidhu85** [comment]
> This looks great, don't have much to say.

Thanks :)

**Pull request merged**